delfin/drivers/dell_emc/vmax/client.py

```diff
@@ -61,11 +61,11 @@ def init_connection(self, access_info):
             msg = ("Failed to connect to VMAX. Host or Port is not correct: "
                    "{}".format(err))
             LOG.error(msg)
-            raise exception.InvalidIpOrPort()
+            raise exception.StorageBackendException(msg)
```

Comment:
The StorageBackendException is a generic exception and will not convey much information to user for the failure reason. It is better to use specific exception whenever possible.

Reply:
done